lib/core/src/sdk.rs

```diff
@@ -875,15 +875,29 @@ impl LiquidSdk {
             .map(|payment| SendPaymentResponse { payment })
     }
 
-    /// Fetch the current limits for Onchain Send swaps
-    pub async fn pay_onchain_limits(&self) -> Result<OnchainPaymentLimitsResponse, PaymentError> {
+    /// Fetch the current limits for Onchain Send and Receive swaps
+    pub async fn fetch_onchain_limits(&self) -> Result<OnchainPaymentLimitsResponse, PaymentError> {
         self.ensure_is_started().await?;
 
-        self.swapper
+        let send_limits = self
+            .swapper
             .get_chain_pairs(Direction::Outgoing)?
             .ok_or(PaymentError::PairsNotFound)
-            .map(|pair| pair.limits)
-            .map(Into::into)
+            .map(|pair| pair.limits)?;
+        let receive_limits = self
+            .swapper
+            .get_chain_pairs(Direction::Incoming)?
+            .ok_or(PaymentError::PairsNotFound)
+            .map(|pair| pair.limits)?;
```

Comment:
This will be two network requests, maybe the swapper function needs changing to return both directions

Reply:
Done: f6477695730baa1bd47f8995a03ce882f80f69e4